config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/InboundJmsDestinationDetail.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.beans;
+
+public class InboundJmsDestinationDetail {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this empty class, write its code or make it an "interface". ([squid:S2094](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2094))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=138&issues=AWdc315wy5Us1olWmrch&open=AWdc315wy5Us1olWmrch)

[ISSUE_KEY]: AWdc315wy5Us1olWmrch